spec/helpers/comments_helper_spec.rb

```diff
@@ -28,20 +28,36 @@
         expect(helper.commenter_id_for_css_classes(comment)).to eq(nil)
       end
     end
+
+    context "when comment is deleted" do
+      let(:comment) { create(:comment, is_deleted: true) }
+
+      it "returns nil" do
+        expect(helper.commenter_id_for_css_classes(comment)).to eq(nil)
+      end
+    end
+
+    context "when comment is hidden by admin" do
+      let(:comment) { create(:comment, hidden_by_admin: true) }
+
+      it "returns nil" do
+        expect(helper.commenter_id_for_css_classes(comment)).to eq(nil)
+      end
+    end
   end
 
   describe "#css_classes_for_comment" do
     context "when comment exists" do
       let(:user) { create(:user) }
       let(:comment) { create(:comment, pseud: user.default_pseud) }
+      let(:unreviewed_comment) { create(:comment, unreviewed: true, pseud: user.default_pseud) }
```

Comment:
We've got an `unreviewed` trait set up in the [comment factory](https://github.com/otwcode/otwarchive/blob/16f204fbbd654dd93634c986b9cedba77058f52b/factories/comments.rb), so this can even be shortened a bit: `create(:comment, :unreviewed, pseud: user.default_pseud) }`.